docs/BreakingChanges/079 ClickOnce supports SHA-256 on 4.0-targeted apps.md

```diff
@@ -0,0 +1,31 @@
+## 79: ClickOnce supports SHA-256 on 4.0-targeted apps
```

Comment:
{
  "Id": "79",
  "Title": "ClickOnce supports SHA-256 on 4.0-targeted apps",
  "Details": "Previously, a ClickOnce app with a certificate signed with SHA-256 would require .NET 4.5 or later to be present, even if the app targeted 4.0. Now, 4.0-targeted ClickOnce apps can run on 4.0, even if signed with SHA-256.",
  "Suggestion": "This change removes that dependency and allows SHA-256 certificates to be used to sign ClickOnce apps that target .NET Framework 4 and earlier versions.",
  "Link": "https://msdn.microsoft.com/en-us/library/dn833125(v=vs.110).aspx#ClickOnce",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.6",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/993689",
  "Notes": "We could possibly detect this by analyzing ClickOnce manifests and looking for SHA-256 certs.",
  "IsRetargeting": true,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis